### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Wednesday, 13th of November 2024, 2:30pm UTC

### DISCUSSION THREAD

**Discussion by t-lo** [flatcar/Flatcar#1560] [Flatcar Office Hours]
In the office hours we discuss news and happenings in and around Flatcar, provide a space for our user community to discuss with maintainers, show demos, and do our release planning.

- Jitsi: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording: https://www.youtube.com/live/7ofguKwR-Co
  - All recordings: https://www.youtube.com/@FlatcarLinux/live
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)
  
## Welcome
- Brief intro/check-in of the new participants joining the call. Please introduce yourself and share what brings you here today.

## Agenda items
- @tuunit discusses IONOS vendor support: ongoing development, status updates, and challenges
- @t-lo gives a brief presentation on the sysext bakery and how you can use it to customise and to extend Flatcar

### Docker 26 is now in Stable!
<details><summary>A number of legacy features have been removed by upstream.</summary>

  [Removed features](https://github.com/docker/cli/blob/master/docs/deprecated.md):
  - [Devicemapper storage driver](https://github.com/docker/cli/blob/master/docs/deprecated.md#device-mapper-storage-driver)
  - [Pushing and pulling with image manifest v2 schema 1](https://github.com/docker/cli/blob/master/docs/deprecated.md#pushing-and-pulling-with-image-manifest-v2-schema-1)
  - [OOM-score adjust for the daemon](https://github.com/docker/cli/blob/master/docs/deprecated.md#oom-score-adjust-for-the-daemon)
  - [logentries logging driver](https://github.com/docker/cli/blob/master/docs/deprecated.md#logentries-logging-driver)
  - [IsAutomated field, and "is-automated" filter on docker search](https://github.com/docker/cli/blob/master/docs/deprecated.md#isautomated-field-and-is-automated-filter-on-docker-search)
  - [Container short ID in network Aliases field](https://github.com/docker/cli/blob/master/docs/deprecated.md#container-short-id-in-network-aliases-field)
  - [legacy API versions](https://github.com/docker/cli/blob/master/docs/deprecated.md#deprecate-legacy-api-versions)
  - [Container and ContainerConfig fields in Image inspect](https://github.com/docker/cli/blob/master/docs/deprecated.md#container-and-containerconfig-fields-in-image-inspect)

</details>

### End of support for CGroupsV1 in early 2025
<details><summary>Flatcar will stop supporting the cgroups v1 backwards compatibility mode in Q1 2025.</summary>

- With our adoption of systemd-256, support for legacy CGroupsV1 will end in **Alpha, Beta, and eventually Stable**.
- [Enabling legacy CGroupsV1 during deployment](https://www.flatcar.org/docs/latest/container-runtimes/switching-to-unified-cgroups/#starting-new-nodes-with-legacy-cgroups) will not be supported anymore.
- Nodes that use CGroupsV1 legacy mode will *fail to update*. This ensures your workloads will not be disrupted. Enable CGroupsV2 on your legacy CGroupsV1 nodes in order to successfully update.
- LTS-2024 will support CGroupsV1 until late 2025.
</details>

### Release Planning

- https://github.com/orgs/flatcar/projects/7/views/8

### Administrative tasks after meeting concluded
- [x] Create a new Github discussion for the next Flatcar Office Hours: https://github.com/flatcar/Flatcar/discussions/new?category=flatcar-office-hours
- [x] Close this discussion as "Resolved".

**Comment by tuunit** [comment]
Hi @t-lo,

@tormath1 asked me if I would be willing to demo the work we've done to get IONOS Cloud support going. If it is fine with you I would like to cut out 5-10 minutes to showcase the PRs and working image.

**Reply by t-lo** [reply]
Hi @tuunit ,

It would be amazing to have you! We love folks dropping by and sharing their experience as much as we love demos.

**Reply by tuunit** [reply]
See you Wednesday then!

**Reply by t-lo** [reply]
Added to the agenda.

**Comment by t-lo** [comment]
Following input I received from a Flatcar user I'd like to present a brief walk-through of using the [sysext-bakery](https://github.com/flatcar/sysext-bakery), if there's time left after the IONOS demo.

**Reply by tuunit** [reply]
I would be highly interested in this as well!
